admin/c2cgeoportal_admin/templates/404.jinja2

```diff
@@ -24,8 +24,14 @@
 {% extends "layout.jinja2" %}
 
 {% block content %}
-<div class="content">
-  <h1><span class="font-semi-bold">Pyramid</span> <span class="smaller">Alchemy scaffold</span></h1>
-  <p class="lead"><span class="font-semi-bold">404</span> Page Not Found</p>
+<div class="container-fluid">
+{% for table in request.c2cgeoform_application.tables() %}
+{% if table["key"] == request.matchdict["table"] %}
+<h2>
+{{ table["title"] }}
+</h2>
+{% endif %}
+{% endfor %}
+<p>{{ _("Element id '{}' not found, probably deleted.").format(request.matchdict["id"]) }}</p>
```

Comment:
I purpose another solution, but I never see something in `context.detail`...

Reply:
Because we never put anything in `HTTPNotFound(detail=...")`

Note that here context is the HTTPNotFound instance.

But it would be good to give the detail when raising the exception than trying to deduce the cause in the template.